pkg/programkind/programkind.go

```diff
@@ -68,6 +69,7 @@ var supportedKind = map[string]string{
 	"yaml":    "",
 	"yara":    "",
 	"yml":     "",
+	"zlib":    "application/zlib",
```

Comment:
is this file extension intentional? I figured it would be "Z"?

Reply:
It is. It's more of a workaround to leverage the header inspection we do.

Otherwise I was going to read the header separately to check the first two bytes.

Reply:
Should we list the supportedKind as 'Z' though, so that it can also go by file extension (filename.tar.Z) as an optimization?

If not, add a comment as to why this isn't using a known extension.

Reply:
Yep, that works for me.

Reply:
Updated in [`f9f4cfa` (#713)](https://github.com/chainguard-dev/malcontent/pull/713/commits/f9f4cfa196902580a570913a97ddbff3bb15606c).